src/workshops/react-refactor-classes/starter-files/workshop/Pokemon.jsx

```diff
@@ -1,6 +1,14 @@
 import React from "react";
-import { getPokemon } from "./utils";
```

Comment:
this was causing tests to fail

Reply:
Weird how this hasn't been an issue for prior cohorts. Don't think I've touched this workshops for ages